<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lecture started</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 234, 228);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="running_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>16</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(33, 82, 115);
font: 63 16pt &quot;Montserrat SemiBold&quot;;
</string>
    </property>
    <property name="text">
     <string>Your Lecture is Running Now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(47, 144, 145);
color: rgb(243, 243, 243);
 border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Stop Lecture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_notify_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 234, 228);
border-color: rgb(47, 144, 145);
color: rgb(33, 82, 115);
border-radius: 15px;
border-width: 2px;
border-style:outset;</string>
    </property>
    <property name="text">
     <string>Show Notification</string>
    </property>
   </widget>
   <widget class="QLabel" name="dowload_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(33, 82, 115);
font: 57 10pt &quot;Montserrat Medium&quot;;

</string>
    </property>
    <property name="text">
     <string>Download Attendance Sheet ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_btn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 216, 216);color: rgb(99, 99, 99);border-radius: 15px;border-style:outset;border-color:  rgb(99, 99, 99);</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="c6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>290</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Group 3.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>541</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Rectangle 1.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="c4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>-40</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>rings2.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="c5">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>180</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-5</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Group 1.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>c4</zorder>
   <zorder>label_3</zorder>
   <zorder>running_label</zorder>
   <zorder>stop_btn</zorder>
   <zorder>show_notify_btn</zorder>
   <zorder>dowload_label</zorder>
   <zorder>download_btn</zorder>
   <zorder>c6</zorder>
   <zorder>c5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="app_images/cylinder2.qrc"/>
  <include location="app_images/rec1.qrc"/>
  <include location="app_images/circle1.qrc"/>
  <include location="app_images/cylinder.qrc"/>
 </resources>
 <connections/>
</ui>
